src/clients/SparkCore.js

```diff
@@ -68,7 +68,6 @@ const MAX_BINARY_SIZE = 108000; // According to the forums this is the max size.
  */
 class SparkCore extends EventEmitter {
   _cipherStream: ?Duplex = null;
-  _connectionKey: ?string = null;
```

Comment:
connectionKey is redundant..its used only in some console.logs inside sparkCore and in DeviceServer for checking if the device is in devicesByIdMap...